python/vineyard/contrib/ml/tests/test_dali.py

```diff
@@ -20,7 +20,10 @@
 
 import lazy_import
 import pytest
+from pytest_cases import parametrize
```

Comment:
Use `import pytest_cases` and `@pytest_cases.parametrize(...)` anywhere you need it to avoid causing confusion with pytest's parametrize.